gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
This should be in `.gitignore` not `gitignore` (not the `.` noting it's a hidden file) and I would actually recommend you add this to your global `.gitignore` https://help.github.com/articles/ignoring-files/#create-a-global-gitignore